Armstrong_Number/Armstrong_Number.rb

```diff
@@ -0,0 +1,54 @@
+# A positive integer of n digits is called an Armstrong number of order n (order is number of digits) if
+# abcd... = pow(a,n) + pow(b,n) + pow(c,n) + pow(d,n) + .... 
+
+# To calculate the order of the number
+def order(num)
+	# variable to store of the number 
+    count = 0
+	while (num != 0) 
+	    count = count + 1
+		num = num / 10
+	end
+	return count 
+end
+
+# Function to check whether the given number is 
+# Armstrong number or not 
+def Armstrong (num) 
+	count = order(num) 
```

Comment:
Use 4 space indentation.

Reply:
Make this changes throughout the file.

Reply:
Means whole file need to be 4 space indent including function.